app/controllers/users_controller.rb

```diff
@@ -17,87 +18,76 @@ def index
   end
 
   def show
-    if authorize @user
-      render json: {
-        status: { code: 200, message: 'Success.' },
-        data: UserSerializer.new(@user).serializable_hash[:data][:attributes]
-      }
-    end
+    return unless authorize @user
+
+    render_success_response('success.', true)
   end
 
   def update
-    if authorize @user
-      @user.update(user_params)
-      render json: {
-        status: { code: 200, message: 'Successfully updated the profile.' },
-        data: UserSerializer.new(@user).serializable_hash[:data][:attributes]
-      }
-    end
+    return unless authorize @user
+
+    @user.update(user_params)
```

Comment:
@josephelias-rubyians  Need error response if the update didn't happen correctly